src/components/pages/Collect/Collect.js

```diff
@@ -45,6 +45,8 @@ import {
 } from '../../../App/mermaidData/recordProtocolHelpers'
 import { getIsUserReadOnlyForProject } from '../../../App/currentUserProfileHelpers'
 import { PAGE_SIZE_DEFAULT } from '../../../library/constants/constants'
+import { RECORD_STATUS_LABELS } from './collectConstants'
```

Comment:
Are you happy with this approach where constants are kept in different places?

Reply:
Im really either or about it. Was just acting out of habit/auto pilot.  I guess the fact that they are removed makes it harder to read/understand and these ones are unlikely to be reused.